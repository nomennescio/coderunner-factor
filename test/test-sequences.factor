! Copyright 2019 nomennescio

USING: tools.testest ;
IN: tests

: run-tests ( -- )

<{ { 0 } -> { 1 } }>
<{ { { 0 } { 0 } } -> { { 0 } { 1 } } }>

;

MAIN: run-tests

