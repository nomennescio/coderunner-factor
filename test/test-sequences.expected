
<FAILED::>Test Failed : Expected :<:LF:>{ 1 }<:LF:>but got :<:LF:>{ 0 }

<FAILED::>Test Failed : Expected :<:LF:>{ { 0 } { 1 } }<:LF:>but got :<:LF:>{ { 0 } { 0 } }
